### ISSUE TITLE
suspicion error

### ISSUE THREAD

**Issue by syus** [joelverhagen/TorSharp#1]
hi,

please note the mistake in the following function 

```c#
namespace Knapcode.TorSharp.Tools.Tor
{
    public class TorControlClient
    {
        ...
        public void Close()
        {
            if (_tcpClient != null)
            {
                _tcpClient.Close();
                _reader.Dispose();
                _reader.Dispose();
            }
        }
``` 
probably replace last _reader.Dispose by _writer.Dispose

cheers
syus

**Comment by joelverhagen** [comment]
Interesting... I'll look into it. Definitely suspect!

**Comment by joelverhagen** [comment]
It's a good fix. Turns out when the `TcpClient` is closed, the socket is closed (i.e. `FIN ACK` stuff), but it is good to dispose these guys too, just in case.

I've published a new version of the NuGet package (1.3.1) with this fix, along with some other cleanup.
